src/components/LineChart/LineChart.jsx

```diff
@@ -517,6 +531,7 @@ class LineChart extends PureComponent {
       },
     ];
     // Append annotation to the graph
+    // eslint-disable-next-line import/no-named-as-default-member
```

Comment:
Would fixing this lint error involve redux? Is that why it is being disabled?

Reply:
I did this because when I tried separating out this logic, nothing was being rendered. I assumed it was maybe a svgAnnotation thing so I added the eslint-diable. Not sure how to fix this otherwise.

Reply:
This seems like something that should be addressed.